src/gondi.ts

```diff
@@ -808,32 +808,72 @@ export class Gondi {
     };
   }
 
-  async liquidateLoan(loan: model.Loan & { loanId: bigint }) {
-    const txHash = await this.contracts.MultiSourceLoanV4.write.liquidateLoan([
-      loan.loanId,
```

Comment:
Oh this is a bug, right? :/
Right now platform will try to liquidate an incorrect loan when liquidation time comes

Reply:
No no, but this version using the first tranche is better and doesn't require an additional loanId param